lib/Cake/Cache/CacheEngine.php

```diff
@@ -181,7 +181,7 @@ public function key($key) {
 
 		$prefix = '';
 		if (!empty($this->_groupPrefix)) {
-			$prefix = vsprintf($this->_groupPrefix, $this->groups());
+			$prefix = md5(implode('_', $this->groups()));
```

Comment:
You just dropped the usage of `_groupPrefix` property?

Reply:
That property contained something like `%s_%s_%s` if the configuration had 3 groups. I don't think there is any actual difference with the current implementation other than there are fewer calls.

We should however, remove the population of this property as it is now unused.

Reply:
`_groupPrefix` seems still to be used in FileEngine.

Reply:
Darn. So close.